app/actions.cpp

```diff
@@ -1,49 +1,39 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 // included header files
-#include "config.h"
-
-#include "app_utils.hpp"
 #include "actions.hpp"
-#include "exiv2app.hpp"
 
-#include "image.hpp"
-#include "jpgimage.hpp"
-#include "xmpsidecar.hpp"
-#include "types.hpp"
-#include "exif.hpp"
+#include "app_utils.hpp"
+#include "config.h"
 #include "easyaccess.hpp"
+#include "exif.hpp"
+#include "futils.hpp"
+#include "i18n.h"  // NLS support.
+#include "image.hpp"
 #include "iptc.hpp"
-#include "xmp_exiv2.hpp"
 #include "preview.hpp"
-#include "futils.hpp"
-#include "i18n.h"                // NLS support.
+#include "types.hpp"
+#include "xmp_exiv2.hpp"
 
 // + standard includes
+#include <sys/stat.h>   // for stat()
+#include <sys/types.h>  // for stat()
+
 #include <filesystem>
-#include <string>
-#include <iostream>
-#include <iomanip>
 #include <fstream>
-#include <sstream>
-#include <cstring>
-#include <cstdio>
-#include <ctime>
-#include <cmath>
-#include <cassert>
+#include <iomanip>
+#include <iostream>
 #include <mutex>
-#include <stdexcept>
-#include <sys/types.h>                  // for stat()
-#include <sys/stat.h>                   // for stat()
+#include <sstream>
 #ifdef EXV_HAVE_UNISTD_H
-# include <unistd.h>                    // for stat()
+#include <unistd.h>  // for stat()
 #endif
 
 #if defined(_WIN32) || defined(__CYGWIN__)
-# include <sys/utime.h>
 #include <Windows.h>
```

Comment:
I thought windows.h was supposed to be lowercase.

Reply:
Yes, but then the NTFS filesystem is case-insensitive by default :sweat_smile: . Anyways, by googling it, I noticed that recently it is possible to enable the case sensitivity on Windows ... so I will correct this.

Reply:
Btw, on MSYS2 (and I guess any other MinGW-w64 distribution), it really is `windows.h` :/

Reply:
hum ... right ... somehow our CI was passing with MinGW & Cygwin ... I do not know if I want to dig there :sweat_smile: